packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -227,64 +229,72 @@ $isPaddedMargin: 24px;
 }
 
 .borderWhileDragging {
-  border-style : solid;
+  border-style: solid;
   border-width: 1px;
-  border-color:$ms-color-themePrimary;
+  border-color: $ms-color-themePrimary;
   -webkit-animation: fadeOut 0.2s forwards;
   animation: fadeOut 0.2s forwards;
 }
 
 .dropHintCircleStyle {
-  display:inline-block;
-  visibility:hidden;
+  display: inline-block;
+  visibility: hidden;
   position: absolute;
   bottom: 0;
   top: -27px;
   height: 9px;
   width: 9px;
   border-radius: 50%;
   @include ms-margin-left(-5px);
-  top:34px;
+  top: 34px;
   overflow: visible;
   z-index: 10;
-  border:1px solid $ms-color-themePrimary;
+  border: 1px solid $ms-color-themePrimary;
   background: $ms-color-white;
 }
 
-.dropHintLineStyle{
+.dropHintLineStyle {
   display: inline-block;
-  visibility:hidden;
+  visibility: hidden;
   position: absolute;
   bottom: 0;
   top: -3px;
   overflow: hidden;
   height: 37px;
   width: 1px;
   background: $ms-color-themePrimary;
-  z-index:10;
+  z-index: 10;
 }
 
-.dropHintStyle{
+.dropHintStyle {
   display: inline-block;
   position: absolute;
 }
 
 .borderAfterDropping {
-  border-style : solid;
+  border-style: solid;
   border-width: 1px;
-  border-color:$ms-color-themePrimary;
+  border-color: $ms-color-themePrimary;
   -webkit-animation: fadeOut 1.5s forwards;
   animation: fadeOut 1.5s forwards;
   @include ms-left(-1px);
   line-height: 31px;
 }
 
-  @-webkit-keyframes fadeOut {
-    from {border-color:$ms-color-themePrimary; }
-    to {border-color:white }
+@-webkit-keyframes fadeOut {
+  from {
+    border-color: $ms-color-themePrimary;
+  }
+  to {
+    border-color: white;
```

Comment:
this should be $ms-color-white I presume?

Reply:
I'm not modifying this file's content - it's changed because of prettier :( I'll redo this with theme.palette in my next part

Reply:
I'll be moving away from sass on the next commit for header